src/components/Profile/components/EmergencyInfoList/index.js

```diff
@@ -0,0 +1,94 @@
+import { useState, useEffect } from 'react';
+import './index.css';
+import ProfileInfoListItem from '../ProfileInfoListItem';
+import user from 'services/user';
+
+import { Typography, Grid, Card, CardHeader, CardContent, List } from '@material-ui/core';
+
+const formatPhone = (phone) => {
+  let tele = String(phone);
+  if (tele.length === 10) {
+    return '(' + tele.slice(0, 3) + ') ' + tele.slice(3, 6) + '-' + tele.slice(6);
+  } else if (tele !== 'undefined') {
+    return tele;
+  }
```

Comment:
The call to `String(phone)` was to make sure that calls to `tele.length` didn't return an error, I believe. However, new(ish) JavaScript syntax makes that much easier to handle:
```suggestion
  if (phone?.length === 10) {
    return `(${phone?.slice(0, 3)}) ${phone?.slice(3, 6)}-${phone?.slice(6)}`;
  } else {
    return phone
  } 
```